awx/ui_next/src/screens/Job/JobOutput/PageControls.jsx

```diff
@@ -10,26 +10,25 @@ import {
 import styled from 'styled-components';
 
 const Wrapper = styled.div`
-  display: grid;
-  grid-gap: 20px;
-  grid-template-columns: repeat(5, 1fr);
+  display: flex;
+  height: 35px;
+  outline: 1px solid #d7d7d7;
+  width: 100%;
 `;
 
 const Button = styled(PFButton)`
-  &:hover {
```

Comment:
Do we need to lose the hover? In my opinion it looks better with this left in. It also more closely aligns with the set of mockups for this view: https://tower-mockups.testing.ansible.com/patternfly/jobs/jobs-output/

Reply:
@trahman73 wanted to remove it to align with PatternFly's [button behavior](https://www.patternfly.org/v4/documentation/react/components/button#variations)

Reply:
works for me - let's merge this.